docs/resources/Guild.md

```diff
@@ -146,11 +148,12 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 
 ###### Mutable Guild Features
 
-| Features         | Required Permissions | Effects                                                   |
-| ---------------- | -------------------- | --------------------------------------------------------- |
-| COMMUNITY        | Administrator        | Enables Community Features in the guild                   |
-| INVITES_DISABLED | Manage Guild         | Pauses all invites/access to the server                   |
-| DISCOVERABLE     | Administrator*       | Enables discovery in the guild, making it publicly listed |
+| Features            | Required Permissions | Effects                                                   |
+| ------------------- | -------------------- | --------------------------------------------------------- |
+| COMMUNITY           | Administrator        | Enables Community Features in the guild                   |
+| DISCOVERABLE        | Administrator*       | Enables discovery in the guild, making it publicly listed |
+| INVITES_DISABLED    | Manage Guild         | Pauses all invites/access to the server                   |
+| RAID_ALERTS_ENABLED | Manage Guild         | Enables alerts for join raids                             |
```

Comment:
Is the `RAID_ALERTS_ENABLED` feature still used? Now, the `RAID_ALERTS_DISABLED` feature is present when raid alerts are disabled.

Reply:
hm that might be right, looks like servers still have `RAID_ALERTS_ENABLED` but it seems to not be used anymore; can staff clarify this?

Reply:
We switched the behavior to be _opt-out_, so `RAID_ALERTS_DISABLED` is the right thing.

Reply:
@hellsan631 thanks for the clarification! I've updated the PR and replaced the guild feature